File: js/src/lexer/TokenType.js
const Enum = require('../common/Enum')

module.exports = {
    KEYWORD : new Enum("KEYWORD", 1),
    VARIABLE : new Enum("VARIABLE", 2),
    OPERATOR : new Enum("OPERATOR", 3),
    BRACKET: new Enum("BRACKET", 4),
    INTEGER: new Enum("INTEGER", 5),
    FLOAT: new Enum("FLOAT", 6),
    BOOLEAN : new Enum("BOOLEAN", 7),
    STRING : new Enum("STRING", 8)
}

File: TinyScript/unstable/python/lexer/TokenType.py
# -*- coding: UTF-8 -*-
'''=================================================
@Project -> File   ：TinyScript -> TokenType
@IDE    ：PyCharm
@Author ：QiangZiBro
@Date   ：2020/4/12 5:53 下午
@Desc   ：
=================================================='''
from common.Enum import Enum

KEYWORD = Enum("KEYWORD", 1)
VARIABLE = Enum("VARIABLE", 1)
OPERATOR = Enum("OPERATOR", 1)
BRACKET = Enum("BRACKET", 1)
INTEGER = Enum("INTEGER", 1)
FLOAT = Enum("FLOAT", 1)
BOOLEAN = Enum("BOOLEAN", 1)
STRING = Enum("STRING", 1)
